### PR TITLE
Update AboutUs.md and Added Skeletal PPP

### PR DESCRIPTION
**Pull request by Bahamas20**
Added my name,photo for AboutUS.md

### CODE REVIEW

File: docs/team/bahamas20.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+### Project: Salesy
+
+Salesy is a desktop program that helps food vendors in NUS manage details of their clients and suppliers. It is optimised for CLI users so that tasks can be done faster by typing in commands. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+
+Given below are my contributions to the project.
+
+* **New Feature**: to be added soon
+
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=weesiau&breakdown=true)
```
Comment (hauchongtang) [inline docs/team/bahamas20.md:14]: Need change to your own github username
```suggestion
* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=Bahamas20&breakdown=true)
```

File: docs/AboutUs.md [hunk 2]
```diff
@@ -48,12 +48,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: Dev Ops + Threading
 
-### James Doe
+### Mohamed Hamas
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/hamas.png" width="200px">
```
Comment (hauchongtang) [inline docs/AboutUs.md:53]: The profile image filename might wanna change to your github username too, for the dashboard to capture

**Review by hauchongtang — CHANGES_REQUESTED** [review]
# Proposed Changes
- profile image filename -> <github username>.png (I think grading script need to capture it)
- reposense link to change to your github username

File: docs/team/bahamas20.md (hunk 1 above)
Comment (Bahamas20) [reply docs/team/bahamas20.md]: Ok got it

File: docs/AboutUs.md (hunk 2 above)
Comment (Bahamas20) [reply docs/AboutUs.md]: Rectified this too

**Review by hauchongtang — APPROVED** [review]

**Review by hauchongtang — APPROVED** [review]
lgtm

**Pull request merged**